tests/parsers/test_77.py

```diff
@@ -37,6 +38,7 @@ def test_lscpu():
 	}
 	output = read_lscpu.read_lscpu(os.path.join(filedir, 'lscpu.txt'))
 
+	#assert output == expect
```

Comment:
What is the error if this line is uncommented?

Reply:
Well, no assert, no error of course. As you can see line 41 wasn't even there, so somebody before me decided that the test should not be performed. In fact, if you do that check it fails. Considering the fact that's the only test that fails and that the output is significally different from the expected one it may well be a wrong expected result. However I didn't felt so confident to simply change the test.

Reply:
@lvps do you know what happened here? Any hint for how to proceed?